src/types/UiaTextRangeBase.cpp

```diff
@@ -251,14 +217,13 @@ const Endpoint UiaTextRangeBase::GetEnd() const noexcept
 // - true if range is degenerate, false otherwise.
 const bool UiaTextRangeBase::IsDegenerate() const noexcept
 {
-    return _degenerate;
+    return _start == _end;
 }
 
-void UiaTextRangeBase::SetRangeValues(const Endpoint start, const Endpoint end, const bool isDegenerate) noexcept
+void UiaTextRangeBase::SetEndpoints(const COORD start, const COORD end) noexcept

```

Comment:
This can make a range where start < end. Is that acceptable? It is impossible to know which to set them to, however.

Reply:
nobody seems to call SetEndpoints in this file (?)

Reply:
Good catch.